shared/chat/conversation/bot/install.tsx

```diff
@@ -64,58 +64,46 @@ const InstallBotPopup = (props: Props) => {
   const [installWithRestrict, setInstallWithRestrict] = React.useState(true)
   const [installInConvs, setInstallInConvs] = React.useState<string[]>([])
   const [disableDone, setDisableDone] = React.useState(false)
-  const {
-    commands,
-    featured,
-    inTeam,
-    inTeamUnrestricted,
-    isBot,
-    readOnly,
-    settings,
-    teamID,
-    teamName,
-  } = Container.useSelector((state: Container.TypedState) => {
-    let inTeam: boolean | undefined
-    let isBot: boolean | undefined
-    let teamRole: TeamTypes.TeamRoleType | null | undefined
-    let teamName: string | null | undefined
-    let teamID: TeamTypes.TeamID = TeamTypes.noTeamID
-    let readOnly = false
```

Comment:
this takes this mega useSelector and breaks it up so it doesn't entirely thrash. please look at this one carefully

Reply:
Looks like the types here could be cleaned up a bunch (not saying you should) but I couldn't find any logic differences here. Bot install seems to work fine :+1: